benchadapt/python/benchadapt/result.py

```diff
@@ -78,7 +78,7 @@ def to_publishable_dict(self):
                 "Result not publishable! `machine_info` xor `cluster_info` must be specified"
             )
 
-        if bool(res_dict["stats"]) != bool(res_dict["error"]):
```

Comment:
Do we have a test that covers this?

Reply:
Oops, there's not even a file that tests it at all directly. Added.